mypy/errors.py

```diff
@@ -213,6 +213,38 @@ def filtered_errors(self) -> list[ErrorInfo]:
         return self._filtered
 
 
+_UniqueErrorT = tuple[codes.ErrorCode, str, int, int, str, str]
+
+
+def _to_unique_error(e: ErrorInfo) -> _UniqueErrorT:
+    return e.code, e.file, e.line, e.column, e.severity, e.message
```

Comment:
This is _somewhat_ duplicating what `duplicate_errors` does, but the latter works with `ErrorTuple`s and has this code inline.